assets/src/components/activities/common/choices/delivery/ChoicesDelivery.tsx

```diff
@@ -49,7 +49,7 @@ export const ChoicesDelivery: React.FC<Props> = ({
         >
           <div className={styles.choicesChoiceWrapper}>
             <label className={styles.choicesChoiceLabel} htmlFor={`choice-${index}`}>
-              <div className="d-flex align-items-center">
+              <div className="d-flex align-items-center col">
```

Comment:
`col` here causes this part to grow in the flex layout `flex-grow:1` so any % based width children start laying out correctly.